BACKEND/userAuth/src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 server:
-  port: 8001
+  port: 5000
```

Comment:
Can you define the port 8001 in a dev profile. or Send email to frontend developers that port has changed from 8001 to 5000. otherwise the dev environment is not going to work.